spring-cloud-stream-reactive/src/test/java/org/springframework/cloud/stream/reactive/StreamListenerReactiveReturnWithFailureTests.java

```diff
@@ -42,6 +40,8 @@
 import org.springframework.messaging.handler.annotation.SendTo;
 import org.springframework.messaging.support.MessageBuilder;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
The Spring Framework Code Style guidelines has static imports at the end: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#import-statements

The intent is ban all static imports bar from JUnit and AssertJ assertions.